drivers/gpio/gpio_nrfx.c

```
@@ -192,40 +68,96 @@ static int gpio_nrfx_config(const struct device *port,
 		break;
 
 	default:
-		return -EINVAL;
+		drive = NRF_GPIO_PIN_D0H1;
+		break;
```

Why not return an error in this case like other drivers do?